index.html

```diff
@@ -154,34 +139,33 @@ <h4>Multi-Purpose Stories</h4>
                         </div>
                         <button type="button">See Project</button>
                     </div>
-                    <div class="rect"></div>
                 </div>
             </div>
         </div>
     </section>
-
-    <section class="contact">
+    <section class="contact" id="Contact-section">
         <div class="contact-text">
             <h5 class="chat">Lets's Chat!</h5>
             <h5 class="mobile">CONTACT US</h5>
-            <p>If you have an application you are interested in developing,
-                a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+            </p>
         </div>
-        <form action="contact" class="contact-form">
+        <form action="https://formspree.io/f/mayadbao" method="POST">
             <div class="data">
-                <input type="text" class="contact-element" name="user-fname" placeholder="First Name"><br class="custom">
-                <input type="text" class="contact-element" name="user-lname" placeholder="Last Name">
+                <input type="text" class="contact-element" name="user-f&lname" placeholder="First and Last Name">
```

Comment:
- [ ] Kindly ensure that the Name input (first name/ last name field ) has an attribute `maxlength="30"`.

Reply:
Well resolved ✅